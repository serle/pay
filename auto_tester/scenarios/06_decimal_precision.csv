type,client,tx,amount
deposit,1,1,1.1
deposit,1,2,1.12
deposit,1,3,1.123
deposit,1,4,1.1234
