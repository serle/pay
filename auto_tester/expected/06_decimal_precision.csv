client,available,held,total,locked
1,4.4664,0.0000,4.4664,false
